extensions/Qdrant/Qdrant/QdrantMemory.cs

```diff
@@ -250,24 +261,30 @@ public async Task DeleteAsync(
 
         try
         {
-            QdrantPoint<DefaultQdrantPayload>? existingPoint = await this._qdrantClient
-                .GetVectorByPayloadIdAsync(index, record.Id, cancellationToken: cancellationToken)
+            IReadOnlyList<RetrievedPoint> existingPoints = await this._qdrantClient
+                .RetrieveAsync(index, new Guid(record.Id), cancellationToken: cancellationToken)
```

Comment:
Merged https://github.com/qdrant/qdrant-dotnet/pull/93 to make this more ergonomic